playwright-tests/pages/base.page.ts

```diff
@@ -52,4 +80,40 @@ export class BasePage {
       ).toBeVisible();
     });
   }
+
+  async verifySocialLinkNavigation(socialPlatform: string, expectedURL: string) {
+    await test.step(`Verify ${socialPlatform} link navigation`, async () => {
+      const locator = this.footerSocialLinks[socialPlatform];
+      await expect(locator).toBeVisible();
+      
+      const href = await locator.getAttribute('href');
+      
+      // For email/slack, just verify href
+      if (socialPlatform === 'slack' || socialPlatform === 'email') {
+        expect(href).toBe(expectedURL);
+        return; 
+      }
+      
+      //const expectedHost = new URL(expectedURL).hostname;
+
+      // Navigate and verify
+      const [newPage] = await Promise.all([
+        this.page.context().waitForEvent('page', { timeout: 2000 }).catch(() => null),
+        locator.click()
+      ]);
+      
+      if (newPage) {
+        expect(newPage.url()).toContain(expectedURL);
```

Comment:
Do you want to verify exact match of the URL? If so, it is better to replace with another assertion - `.toHaveURL()`. 
Docs: https://playwright.dev/docs/api/class-pageassertions#page-assertions-to-have-url